releases_test.go

```diff
@@ -0,0 +1,343 @@
+package gitlab
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+)
+
+const exampleReleaseListRsp = `[
```

Comment:
I think it would improve readability if we simplify these examples do they are more concise. For example the descriptions can be just one or two words instead of an actual full description.